### PR TITLE
Added example application that builds the HTML version of the changelog for the Joomla Platform.

### PR DESCRIPTION
**Pull request by eddieajau**
This example shows some usage of the JHttp class as well as adding a supporting markdown parser class for the descriptions of the Github pull requests (highlighting the usage of JLoader::register).

### CODE REVIEW

**Comment by ianmacl** [comment]
# Test Results

Total Tests: 1465
Assertions: 10002
Failures: 0
Errors: 0
Test Time: 195.227067

**Comment by ianmacl** [comment]
# Test Results

Total Tests: 1465
Assertions: 10002
Failures: 0
Errors: 0
Test Time: 203.597132

File: examples/applications/changelog/run.php [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+// Register the markdown parser class so it's loaded when needed.
+JLoader::register('ElephantMarkdown', __DIR__.'/includes/markdown.php');
+
+/**
+ * An example command line application class.
+ *
+ * This application builds the HTML version of the Joomla Platform change log from the Github API
+ * that is used in news annoucements.
+ *
+ * @package		NewLifeInIT
```
Comment (chdemko) [inline examples/applications/changelog/run.php:38]: Should it be 
@package  Joomla.Examples
?

**Comment by eddieajau** [comment]
Moved examples to a new home: https://github.com/joomla/joomla-platform-examples

**Pull request closed without merging**